### PR TITLE
Pull request template README

### PR DESCRIPTION
**Pull request by jlmaurer**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
```
Comment (jlmaurer) [inline .github/pull_request_template.md:4]:
```suggestion
<!--- Describe your changes in detail -->
<!--- When creating a new PR, make sure to request a review and tag related issues -->
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<!--- When creating a new PR, make sure to request a review and tag related issues -->
```
Comment (Askaholic) [inline .github/pull_request_template.md:5]: I think you need to be a contributor in order to be able to request reviews, so it's not something that anyone can do.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
```
Comment (Askaholic) [inline .github/pull_request_template.md:9]: You are already requesting issues to be linked in the description section. I would only link issues in one place.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
+- [ ] Have you written new tests for your core changes, as applicable?
+- [ ] Have you successfully ran tests with your changes locally?
```
Comment (Askaholic) [inline .github/pull_request_template.md:38]: I'd probably make these "I have" statements like the ones below, and not questions.

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] My code is consistent with the code style of this project.
```
Comment (Askaholic) [inline .github/pull_request_template.md:39]: Does this project even have a code style?

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
```
Comment (Askaholic) [inline .github/pull_request_template.md:41]: Could be combined into one point

File: .github/pull_request_template.md (hunk 2 above)
Comment (jlmaurer) [reply .github/pull_request_template.md]: Good point, removed that line.

File: .github/pull_request_template.md (hunk 3 above)
Comment (jlmaurer) [reply .github/pull_request_template.md:8]: Ditto above.

File: .github/pull_request_template.md (hunk 4 above)
Comment (jlmaurer) [reply .github/pull_request_template.md]: Ditto above.

File: .github/pull_request_template.md (hunk 5 above)
Comment (jlmaurer) [reply .github/pull_request_template.md]: Yes I would say so, not in formatting so much but in functionality. But probably not needed.

File: .github/pull_request_template.md (hunk 6 above)
Comment (jlmaurer) [reply .github/pull_request_template.md:39]: @Askaholic I think having two points is actually beneficial here, as one could easily imagine situations where one or the other point would apply.

**Pull request merged**